bindings/experimental/distrdf/python/DistRDF/Ranges.py

```diff
@@ -39,14 +43,11 @@ class EmptySourceRange(DataRange):
     end (int): Ending entry of this range.
     """
 
-    def __init__(self, rangeid, start, end):
-        """set attributes"""
-        self.id = rangeid
-        self.start = start
-        self.end = end
+    start: int
+    end: int
```

Comment:
there is still docs about `id` in this class but the data member was moved to `DataRange`